validator/bundler.py

```diff
@@ -41,13 +41,14 @@ def bundle_datafile_spec(spec):
 
 
 def bundle_resources(resource_dir, thread_pool_size):
-    specs = init_specs(resource_dir)
+    schema_re = re.compile('^\$schema: (?P<schema>.+\.ya?ml)$', re.MULTILINE)
+    specs = zip(init_specs(resource_dir), itertools.repeat(schema_re))
     pool = ThreadPool(thread_pool_size)
-    results = pool.map(bundle_resource_spec, specs)
+    results = pool.starmap(bundle_resource_spec, specs)
```

Comment:
I find this a bit confusing. Perhaps we could define a const `SCHEMA_RE` and avoid defining it as an argument to `bundle_resource_spec`. The fact that it's always the same regular expression makes me think this is unneeded

Reply:
hahaha... That was my first implementation, but global constants are usually frowned upon, so I did this. The fact that it needs to be repeated is due to multiprocessing. Anyway, this can be avoided in this context and will be clearer.

Reply:
fixed